Wikipedia/Code/UserHistoryFunnel.swift

```diff
@@ -43,6 +46,17 @@
         if let readingListCount = try? dataStore.viewContext.allReadingListsCount() {
             event["measure_readinglist_listcount"] = readingListCount
         }
+
+        var feedEnabledList = [String: Any]()
+        [WMFContentGroupKind.featuredArticle, WMFContentGroupKind.topRead, WMFContentGroupKind.onThisDay, WMFContentGroupKind.news, WMFContentGroupKind.relatedPages, WMFContentGroupKind.continueReading, WMFContentGroupKind.location, WMFContentGroupKind.random, WMFContentGroupKind.pictureOfTheDay].forEach({contentGroupKind in
```

Comment:
You could use static `customizableContentGroupKindNumbers` and `globalContentGroupKindNumbers` on `WMFFeedContentController` here, otherwise you'll be missing all the global groups + `WMFContentGroupKindLocationPlaceholder`

Reply:
Users can't disable logging WMFContentGroupKindLocationPlaceholder cells separately from WMFContentGroupKindLocation cards.

Reply:
(in `Settings > Explore feed`)

Reply:
though we can probably log the placeholder cards in `MobileWikiAppiOSFeed` vs this schema

Reply:
both content groups are disabled when either of them is disabled, the advantage of using the static arrays on `WMFExploreFeedContentController` instead of redeclaring it here would be not having to remember to update them in 2 places if anything changes

Reply:
(also, users can disable `WMFContentGroupKindLocationPlaceholder` directly from the feed)